packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleSelectTests.cs

```diff
@@ -77,6 +77,23 @@ public void DropdownAppearance_AttributeIsSet(DropdownAppearance value, string e
         Assert.Contains(expectedAttribute, select.Markup);
     }
 
+    [Fact]
+    public void Select_FilterModeStandardIsSet()
+    {
+        var expectedContents = "filter-mode=\"standard\"";
+        var breadcrumb = RenderWithPropertySet(x => x.FilterMode, FilterMode.Standard);
+
+        Assert.Contains(expectedContents, breadcrumb.Markup);
+    }
+
+    [Fact]
+    public void Select_FilterModeNoneIsSet()
+    {
+        var breadcrumb = RenderWithPropertySet(x => x.FilterMode, FilterMode.None);
+
+        Assert.DoesNotContain("filter-mode", breadcrumb.Markup);
+    }
```

Comment:
```suggestion
    public void Select_FilterModeStandardIsSet()
    {
        var expectedContents = "filter-mode=\"standard\"";
        var select = RenderWithPropertySet(x => x.FilterMode, FilterMode.Standard);

        Assert.Contains(expectedContents, select.Markup);
    }

    [Fact]
    public void Select_FilterModeNoneIsSet()
    {
        var select= RenderWithPropertySet(x => x.FilterMode, FilterMode.None);

        Assert.DoesNotContain("filter-mode", select.Markup);
    }
```

Reply:
Of _course_ I left some copy-pasta! GAH! 🤦‍♂️